trusted-committer/de/02-ensuring-product-quality-de.asciidoc

```diff
@@ -0,0 +1,39 @@
+Wir alle wissen, wie schnell dieses Vertrauen durch eine einzige fehlerhafte Auslieferung ersch�ttert werden kann.
+
+_Trusted Committer_ stellen au�erdem sicher, das in der Organisation die notwendige Infrastruktur und die notwendigen Tools vorhanden sind, welche zur Entwicklung qualitativ hochwertiger Software ben�tigt werden.
+Zur Sicherstellung der Qualit�t wird am h�ufigsten das _Peer Review_ genutzt, welches normalerweise Bestandteil der Pull Requests(PR's) ist.
+W�hrend f�r gew�hnlich jedermann Pull Requests starten und daran teilnehmen kann indem er auf notwendige Verbesserungen hinweist, ist es normalerweise nur dem _Trusted Committer_ m�glich Beitr�ge zu akzeptieren oder abzulehnen.
+Das ist es, was wir an anderer Stelle damit gemeint haben, wenn wir davon sprachen, dass "_Trusted Committer_ n�her an der Freigabe des Codes sind". 
+_Trusted Committer_ sollten den https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] auch w�hrend eines PR's helfen ihre Beitr�ge fertig zu stellen.
+
+Dennoch ist es letztendlich die Aufgabe des _Contributors_, dies zu erreichen.
+Es ist nicht Aufgabe eines _Trusted Committer_, alle Beitr�ge grunds�tzlich zu akzeptieren, sondern nur diejenigen, die die definierten Kriterien bez�glich Qualit�t, Umfang und Projektausrichtung erf�llen. 
+_Trusted Committer_ sollten vermeiden, den Code eines _Contributors_ umzuschreiben, um ihn passend zu machen, selbst wenn dadurch ein h�herer Aufwand bei der Betreuung des https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] w�hrend eines PR's entsteht. 
+_Trusted Committer_ nehmen eine l�ngerfristige Perspektive ein und verstehen, dass diese Art der Unterst�tzung eine Investition in die Langlebigkeit der Organisation darstellt und dies langfristig die Entwicklungsgeschwindigkeit der Organisation erh�ht.
+
+Manchmal sind Anforderungen und Einschr�nkungen des Projekts nicht von vorne herein bekannt, sondern werden erst w�hrend der Entwicklung erkannt.
+_Trusted Committer_ sind auch daf�r verantwortlich, dass diese Ergebnisse erfasst und dokumentiert werden, so dass diese sowohl den https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/product-owner/01-opening-article.asciidoc[_Product Owners_] als auch den https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] zug�nglich sind.
+
+Der Aufgabenbereich des _Trusted Committer_ in Bezug auf Qualit�t geht �ber Pull Requests hinaus. 
+_Trusted Committer_ betrachten Qualit�t als strategische Komponente und stellen die Langlebigkeit der zu erstellenden Software sicher. Dies beinhaltet codeorientierte Verantwortlichkeiten im Sinne von _clean code_ bis hin zur Wahrung der konzeptionellen Integrit�t der gesamten Software.
```

Comment:
I'm not sure, the original text of "cleanliness of the code" should be translated to "clean code" - the latter one means a very specific type of understanding of what cleanliness of the code means that I'm not sure we want to push for here. Maybe translate with something like "Sauberkeit des Quellcodes" or "Verständlichkeit des Quellcodes"